app/views/patients/_abortion_information.html.erb

```diff
@@ -66,16 +66,14 @@
         <!-- Future right of math -->
       </div>
 
-      <div class="col-sm-6">
-        <div id="existing-external-pledges">
-          <%# slight hack - individual edit forms for each pledge %>
-          <% patient.external_pledges.each do |external_pledge| %>
-            <%= render 'external_pledges/edit', patient: patient, external_pledge: external_pledge %>
-          <% end %>
-        </div>
+        <%= render partial: 'external_pledges/external_pledges_table', locals: { patient: patient, external_pledges: patient.external_pledges.order_by('created_at desc') } %>
 
         <div id="new-external-pledge">
-          <%= render 'external_pledges/new', patient: patient, external_pledge: new_external_pledge %>
+          <%= bootstrap_form_for [patient, new_external_pledge], remote: true do |f| %>
```

Comment:
Thoughts about keeping this in the `external_pledges/_new` partial? I think it might be slightly more findable there